libp2p/protocols/pubsub/floodsub.nim

```diff
@@ -106,7 +106,7 @@ method rpcHandler*(f: FloodSub, peer: PubSubPeer, data: seq[byte]) {.async.} =
     debug "failed to decode msg from peer", peer, err = error
     raise newException(CatchableError, "Peer msg couldn't be decoded")
 
-  trace "decoded msg from peer", peer, msg = rpcMsg.shortLog
+  trace "decoded msg from peer", peer, payload = rpcMsg.shortLog
```

Comment:
msgShortLog?

Reply:
Kind of a similar reasoning: `payload` is a widely accepted word for this situations.

Reply:
Kind of the same reasoning, `payload` is a widely accepted word for this situations.

Reply:
This is not logging the whole content https://github.com/vacp2p/nim-libp2p/blob/96bfefc928ee06924a4e97cd8910dd541c0cf328/libp2p/utility.nim#L24

Reply:
Would that be an issue?

Reply:
`shortLog` creates a shortened representation of the data, it's not the payload. What about `summary`?

Reply:
`shortLog` is actually a bit of an abused name, as it vastly depends on the object being shortlogged: Some are actual short representations, some others are obfuscated representations, and others are the full object representations. 

What I mean is: There's not a unique key that perfectly handles all cases; unless we go for a very generic `context` or the likes, which doesn't really provide meaning. Ideally it should be handled on a case by case basis, but I think that's a different task altogether.

Personally, I'd stick to `payload` as it's already there and we can get it merged already, and create a chore to keep track of this naming issue and spend on time whenever we feel appropriate. That is, unless you strongly oppose it.

Reply:
Ok, could you please create an issue?

Reply:
Done: https://github.com/vacp2p/nim-libp2p/issues/1181